src/backend/gl/Cargo.toml

```diff
@@ -17,29 +17,21 @@ build = "build.rs"
 name = "gfx_backend_gl"
 
 [features]
-#default = ["wgl", "surfman", "surfman-x11"]
-default = []
-wgl = ["winapi"]
-#surfman-x11 = ["surfman/sm-x11"]
+x11 = ["surfman/sm-x11"]
 
 [dependencies]
 arrayvec = "0.5"
 bitflags = "1"
-log = { version = "0.4" }
-gfx-hal = { path = "../../hal", version = "0.6" }
+log = "0.4"
+hal = { path = "../../hal", version = "0.6", package = "gfx-hal" }
 auxil = { path = "../../auxil/auxil", version = "0.7", package = "gfx-auxil", features = ["spirv_cross"] }
 smallvec = "1.0"
-glow = "0.4"
+glow = "0.6.1"
 parking_lot = "0.11"
 spirv_cross = { version = "0.22", features = ["glsl"] }
 lazy_static = "1"
 raw-window-handle = "0.3"
-# disable this for now, the versions of glutin are jumping too fast
-#glutin = { version = "0.24", optional = true }
-
-#[target.'cfg(all(unix, not(target_os = "ios")))'.dependencies]
-# TODO: Update to released version when it comes out
-#surfman = { git = "https://github.com/servo/surfman.git", rev = "41ac1ee", features = ["sm-raw-window-handle"], optional = true }
+surfman = { version = "0.3", features = ["sm-raw-window-handle"] }
```

Comment:
I'm surprised this compiles when targeting WASM platform

Reply:
Hmm, it doesn't look like it's because of surfman, but it won't compile for WASM:

```
error: The platform you're compiling for is not supported by winit
  --> /home/zicklag/.cargo/registry/src/github.com-1ecc6299db9ec823/winit-0.19.3/src/platform/mod.rs:26:1
   |
26 | compile_error!("The platform you're compiling for is not supported by winit");
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```